py/estimator_runconfig_tests.py

```diff
@@ -56,6 +60,7 @@ def verify_runconfig(master_host, namespace, job_name, replica, num_ps,
     "chief": chief_list,
     "ps": ps_list,
     "worker": worker_list,
+    "evaluator": evaluator_list,
```

Comment:
Is the evaluator listed as part of the cluster spec? The TFConfig documentation looks different: https://github.com/tensorflow/tensorflow/blob/r1.12/tensorflow/python/estimator/training.py#L415

Reply:
mm, good point.  in this case, presubmit will break.  will ping back when we have result from it.

Reply:
Also does the evaluator_master field get set? https://github.com/tensorflow/estimator/blob/master/tensorflow_estimator/python/estimator/run_config.py#L420

Reply:
no, I don't think so.  this is returned from test server: {"master": "", "task_id": 0, "num_ps_replicas": 0, "num_worker_replicas": 0, "cluster_spec": {}, "task_type": "evaluator", "is_chief": false}